src/Jira/Api.php

```diff
@@ -198,6 +198,16 @@ public function editIssue($issue_key, array $params)
 	}
 
 	/**
+	 * Gets attachment upload limit.
+	 *
+	 * @return array
+	 */
+	public function getAttachmentsMeta()
```

Comment:
Please respect both your and my time. If I say name method like that, then please do name it exactly like it or state your objections, why this can't be done.